traffic_ops/testing/api/assert/assertions.go

```diff
@@ -0,0 +1,109 @@
+package assert
+
+/*
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import (
+	"reflect"
+	"testing"
+)
+
+// Equal asserts that two objects are equal.
+func Equal(t *testing.T, a, b interface{}, msg string) bool {
+	t.Helper()
+	if a != b {
+		t.Error(msg)
```

Comment:
To be a drop-in replacement for the testify library, no, because that wouldn't. Wouldn't bother me if it did though, or if this actually supported variadic arguments for string formatting like `testify` does. It could save some `fmt.Sprintf`-ing in the calls.